data/json/monsters/feral_humans.json

```diff
@@ -354,7 +354,7 @@
     "speed": 85,
     "special_attacks": [ { "id": "feral_weapon_mace" } ],
     "death_drops": "feral_armored_death_drops_mace",
-    "extend": { "weakpoint_sets": [ "wps_humanoid_body_armor", "wps_humanoid_open_helmet" ] },
+    "extend": { "weakpoint_sets": [ "wps_humanoid_body_armor", "wps_humanoid_open_helmet" ], "families": [ "prof_wp_syn_armored" ] },
```

Comment:
I specifically removed those, because they are not wearing synthetic armors. Medieval armor is not the same as kevlar armors that we use for other monsters that use this proficiency. I'd appreciate if you didn't give this back to them.